gulpfile.js

```diff
@@ -155,8 +155,22 @@ gulp.task('karma', function(done) {
   function testMinified() {
     gutil.log('Running unit tests on minified source.');
     buildJs(true);
-    karmaConfig.releaseMode = true;
-    karma.start(karmaConfig, done);
+    process.env['MATERIAL_RELEASE_MODE'] = 'true';
+    karma.start(karmaConfig, testMinifiedJQuery);
+  }
+
+  function testMinifiedJQuery() {
+    gutil.log('Running unit tests on minified source using JQuery.');
+    buildJs(true);
```

Comment:
This second `buildJs` is probably unnecessary since the previous `testMinified()` already built it.